### PR TITLE
Apply fixes from StyleCI

### PR DESCRIPTION
**Pull request by svpernova09**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://github.styleci.io).

---

For more information, click [here](https://github.styleci.io/analyses/zY40ed).

### CODE REVIEW

File: app/src/Event/EventFormType.php [hunk 1]
```diff
@@ -168,26 +167,26 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
                 [
                     'data_class' => null,
-                    'label' => 'Upload new icon',
-                    'required' => false,
-                    'attr'=> [
-                        'class'=>'file',
+                    'label'      => 'Upload new icon',
+                    'required'   => false,
+                    'attr'       => [
+                        'class'=> 'file',
```
Comment (zghosts) [inline app/src/Event/EventFormType.php]: shouldn't there be a space before the => here as well?

Comment (svpernova09) [reply app/src/Event/EventFormType.php]: Probably? But I'm not wanting to hold up the larger change if getting this process automated, then we can tweak the rules to fix things like this. (Or if there aren't rules we can just manually fix)

Comment (zghosts) [reply app/src/Event/EventFormType.php]: Fair enought we can see to fix this later

**Comment by svpernova09** [comment]
I didn't mean to close this, weird. Must have pushed the wrong button >.<

**Pull request merged**